compiler/pota-quantization-value-test/test_fake_wquant.sh

```diff
@@ -55,12 +57,6 @@ while [ "$1" != "" ]; do
       "${TEST_RESULT_FILE}.fake_quantized.circle" \
       --tensors_to_hdf5 "${TEST_RESULT_FILE}.fake_quantized.circle.h5"
 
-    # Compare result
-    "${VIRTUALENV}/bin/python" "${COMPARE_SCRIPT_PATH}" \
-      --input_h5 "${TEST_RESULT_FILE}.fake_quantized.circle.h5" \
-      --expect_dir "${SOURCE_PATH}/expected_outputs/${MODELNAME}/${GRANULARITY}/${DTYPE}/fake_quantization" \
-      --mode fake_quantization
-
     if [[ $? -eq 0 ]]; then
       touch "${PASSED_TAG}"
     fi
```

Comment:
I guess that passed file is generated regardless of the result of comparison?

Reply:
@jinevening I'll remove the flow of pass file creation. Instead, I just print the related logs at the end of the comparing script. Below is an example for the log.

```bash
Start 61: pota_quantization_test
3/7 Test #61: pota_quantization_test ...............***Failed    0.60 sec
-- Found CIRCLE_QUANTIZER: /home/seongwoo/ONE/build/compiler/circle-quantizer/circle-quantizer
-- Found CIRCLE_TENSORDUMP: /home/seongwoo/ONE/build/compiler/circle-tensordump/circle-tensordump
-- Found workdir: /home/seongwoo/ONE/build/compiler/common-artifacts
~/ONE/build/compiler/common-artifacts ~/ONE/build/compiler/pota-quantization-value-test
~/ONE/build/compiler/pota-quantization-value-test
FAILED
- Add_002.channel.int16
ERROR: Quantized scale of ifm1 ([0.00014653]) do not match with expected value (110.00014653272228).

- AveragePool2D_000.channel.int16
ERROR: Quantized zero_point of ifm ([0]) do not match with expected value (110.0).
```